Khan/internal-webserver
# Revert added waf_alert.py

## Summary:
This script does a very simplistic check for WAF attack and notifies us via
slack if it notices anything that looks like a WAF attack.

Issue: INFRA-6453

## Test plan:
- Run "python waf_alert.py"
- Check bot-testing Slack channel to see if the generated alert with % blocked request, IP, Req User Agent, and  WAF Reason with individual %s show up.
